setup.py

```diff
@@ -18,9 +18,12 @@
 from collections.abc import Iterator
 from typing import Any
 
+from pybind11.setup_helpers import ParallelCompile
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
+ParallelCompile("MAX_CONCURRENCY", default=0).install()
```

Comment:
```suggestion
ParallelCompile("MAX_CONCURRENCY").install()
```
Looking at https://github.com/pybind/pybind11/blob/c2b32b1e3b1d972186ff745c9190cad664c219bb/pybind11/setup_helpers.py#L410-L413, `default` is already zero.